HW04/Aliaksandr_Korzun/array.rb

```diff
@@ -0,0 +1,24 @@
+class Array
+  def my_each
+    index = 0
+    while index < length
+      yield self[index]
+      index += 1
+    end
+    self
+  end
+
+  def my_map
+    return_array = []
```

Comment:
Maybe just `array`? 🤔 
```suggestion
    array = []
```